test/utils/preRunChecks.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { preRunChecks } from "../../src/utils";
+import sinon, { stubConstructor, stubInterface, stubObject } from "ts-sinon";
+import { expect } from "chai";
+import {
+  Diagnostic,
+  Extension,
+  Range,
+  Selection,
+  TextDocument,
+  TextEditor,
+} from "vscode";
+import { stub, Stub } from "../utils";
+const tsSinon = { stubConstructor, stubInterface, stubObject };
+
+type FunctionPrarmeters = Parameters<typeof preRunChecks>;
```

Comment:
This type contains a typo and should be renamed to `FunctionParameters` to be named correctly.